src/main/groovy/org/jetbrains/intellij/tasks/RunIdeTask.groovy

```diff
@@ -129,9 +129,7 @@ class RunIdeTask extends JavaExec {
     private void configureClasspath() {
         File ideaDirectory = getIdeaDirectory()
         def executable = getExecutable()
-        if (executable) classpath += project.files("${new File(executable).parent}/../../lib/tools.jar")
-        def toolsJar = Jvm.current().toolsJar
-        if (toolsJar != null) classpath += project.files(toolsJar)
+        if (executable) classpath += project.files("${new File(executable).parent}/../lib/tools.jar")
```

Comment:
What if executable is not set?
Are you sure about the path to tools jar in different jre (JetBrains and oracle) for different oses?

Reply:
Based on this https://docs.oracle.com/javase/8/docs/technotes/tools/windows/jdkfiles.html , it's safe to assume  about the path to `tools.jar`. Looks like jbre is using the same layout.

Reply:
@zolotov done. I've opted out on updating the changelog 😂 .